apps/condo/domains/meter/hooks/useMeterTableColumns.tsx

```diff
@@ -0,0 +1,172 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import get from 'lodash/get'
+import { getDateRender, getTextRender } from '../../common/components/Table/Renders'
+import { useIntl } from 'react-intl'
+import React, { CSSProperties, useCallback, useState } from 'react'
+import { InputNumber } from 'antd'
+import { colors } from '@condo/domains/common/constants/style'
+import { fontSizes } from '@condo/domains/common/constants/style'
+import pickBy from 'lodash/pickBy'
+import { isEmpty } from 'lodash'
+
+const inputNumberCSS = css`
+  & .ant-input-number-handler-wrap {
+    visibility: hidden;
+  }
+
+  font-size: ${fontSizes.label};
+  
+  width: 100%;
+  padding-right: 20%;
+`
+
+const INPUT_CONTAINER_STYLE: CSSProperties = { position: 'relative' }
+const inputMeterReadingFormatter = value => value.toString().replace(',', '.')
+const inputMeterReadingParser = input => input.replace(/,+/g, '.')
+const METER_READING_INPUT_ADDON_STYLE: CSSProperties = {
+    position: 'absolute',
+    top: '50%',
+    right: '2%',
+    padding: '0 5px',
+    fontSize: '0.8rem',
+    backgroundColor: colors.backgroundLightGrey,
+    verticalAlign: 'middle',
+    height: '85%',
+    borderRadius: '0 8px 8px 0',
+    transform: 'translateY(-50%)',
+    width: '30%',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    color: colors.sberGrey[6],
+}
+
+const MeterReadingInput = ({ record, newMeterReadings, setNewMeterReadings }) => {
+    const intl = useIntl()
+    const AddMeterReadingPlaceholderMessage = intl.formatMessage({ id: 'pages.condo.meter.create.AddMeterReadingPlaceholder' })
+
+    const meterReadingValueChangeHandler = useCallback((meterId, tariffNumber, e) => setNewMeterReadings(meterReadings => {
```

Comment:
can you format this please, not very readable.

Reply:
formatted this